### PR TITLE
code normalization and fix PSR-2

### PR DESCRIPTION
**Pull request by lgcosta**

### CODE REVIEW

File: servicegroup-monitoring/src/class/ServicegroupMonitoring.class.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Copyright 2005-2011 MERETHIS
```
Comment (sc979) [inline servicegroup-monitoring/src/class/ServicegroupMonitoring.class.php]:
```suggestion
 * Copyright 2005-2019 Centreon
```

Comment (lgcosta) [reply servicegroup-monitoring/src/class/ServicegroupMonitoring.class.php]: missed file to review. Done!
Thanks

File: servicegroup-monitoring/src/export.php [hunk 2]
```diff
@@ -102,31 +106,41 @@
...
+    1 => "Warning",
+    2 => "Critical",
+    3 => "Unknown",
+    4 => "Pending"
+);
 
-$query = "SELECT SQL_CALC_FOUND_ROWS DISTINCT name ";
-$query .= "FROM servicegroups ";
+$query = "SELECT SQL_CALC_FOUND_ROWS DISTINCT name
+    FROM servicegroups ";
```
Comment (jdelpierre) [inline servicegroup-monitoring/src/export.php]: Maybe, don't use a new line for the "FROM" ?
it must fit on a single line no ?

File: servicegroup-monitoring/src/index.php [hunk 3]
```diff
@@ -72,51 +72,75 @@
...
     2 => "Critical",
     3 => "Unknown",
-    4 => "Pending");
+    4 => "Pending"
+);
 
-$query = "SELECT SQL_CALC_FOUND_ROWS DISTINCT name, servicegroup_id ";
-$query .= "FROM servicegroups ";
+$query = "SELECT SQL_CALC_FOUND_ROWS DISTINCT name, servicegroup_id
+    FROM servicegroups ";
```
Comment (jdelpierre) [inline servicegroup-monitoring/src/index.php]: Maybe, don't use a new line for the "FROM" ?
it must fit on a single line no ?

File: servicegroup-monitoring/src/table.ihtml [hunk 4]
```diff
@@ -41,54 +43,67 @@
                 {elseif $state == 4 }
                     {assign var='link' value='&o=svc_pending'}
                 {/if}
-            <span class="state_badge" style='background-color: {$serviceStateColors[$state]};'></span><span class='margin_right'>{if $displayLink}<a href='/{$centreon_web_path}/{$elem.sgurl}{$link}' target=_blank>{/if}{$counter}{if $displayLink}</a>{/if}</span>
+            <span class="state_badge" style='background-color: {$serviceStateColors[$state]};'></span>
+            <span class='margin_right'>
+                {if $displayLink}
+                <a href='/{$centreon_web_path}/{$elem.sgurl}{$link}' target=_blank>{$counter}</a>
```
Comment (jdelpierre) [inline servicegroup-monitoring/src/table.ihtml]: wrong indentation

File: servicegroup-monitoring/src/table.ihtml [hunk 5]
```diff
@@ -41,54 +43,67 @@
...
+                {else}
+                {$counter}
```
Comment (jdelpierre) [inline servicegroup-monitoring/src/table.ihtml]: wrong indentation

File: servicegroup-monitoring/src/export.php (hunk 2 above)
Comment (lgcosta) [reply servicegroup-monitoring/src/export.php]: Yes ! I agree

Thanks @jdelpierre !

File: servicegroup-monitoring/src/table.ihtml (hunk 5 above)
Comment (lgcosta) [reply servicegroup-monitoring/src/table.ihtml]: Here have two code languages.. I have not found a formal document for template indentation, do we have to indent template code in relation to html?
```
{{ if xpto == zyx }}
<div id="xpto">blablabla</div>
{{ else }}
<div id="xpto">nothing</div>
{{ endif }}
```
or
```
{{ if xpto == zyx }}
    <div id="xpto">blablabla</div>
{{ else }}
    <div id="xpto">nothing</div>
{{ endif }}
```

**Pull request merged**